### PR TITLE
Removed GeneratePackageOnBuild

### PR DESCRIPTION
**Pull request by matmork**
Removed GeneratePackageOnBuild and changed to VersionPrefix so CI builds correctly.

### CODE REVIEW

**Review by hansmbakker — COMMENTED** [review]
Two questions, otherwise it looks good!

File: .github/workflows/nuget-push.yml [hunk 1]
```diff
@@ -51,4 +51,4 @@ jobs:
           dotnet pack -o staging -c Release src/Rive.Maui/Rive.Maui.csproj
 
       - name: Publish
-        run: nuget push staging/*.nupkg -Source 'https://api.nuget.org/v3/index.json' -ApiKey ${{secrets.NUGET_API_KEY}}
+        run: nuget push staging/*.nupkg -Source https://api.nuget.org/v3/index.json -ApiKey ${{secrets.NUGET_API_KEY}} -SkipDuplicate -NonInteractive
```
[the remark this answers is not in the document]
Comment (matmork) [reply .github/workflows/nuget-push.yml:54]: -NonInteractive    Do not prompt for user input or confirmations.
-SkipDuplicate    If a package and version already exists, skip it and continue with the next package in the push, if any.

File: src/Rive.Android/Rive.Android.csproj [hunk 2]
```diff
@@ -4,17 +4,16 @@
 		<SupportedOSPlatformVersion>21</SupportedOSPlatformVersion>
 	</PropertyGroup>
 
-	<PropertyGroup Condition="'$(Configuration)' == 'Release'">
-		<NoWarn>BG8401;BG8605;BG8606;CS0108;CS0114</NoWarn>
```
[the remark this answers is not in the document]
Comment (matmork) [reply src/Rive.Android/Rive.Android.csproj:8]: Not quite sure how to handle this yet, fix them in the binding project or ignore them. They are "safe" to ignore.

**Comment by matmork** [comment]
> Two questions, otherwise it looks good!

Sorry was a bit too quick haha. Wanted to try the Nuget push

**Pull request merged**